server/webdriver/cdc_tests/autopush_test.py

```diff
@@ -66,3 +67,17 @@ def test_statvar_explorer(self):
                                '//*[contains(text(), "Average Annual Wage")]')
     self.assertIsNotNone(custom_sv_elem,
                          'Custom SV (Average Annual Wage) element not found')
+
+
+class CdcAutopushNLTest(unittest.TestCase):
+
+  def test_detect_and_fulfill(self):
+    """Tests that the detect-and-fulfill endpoint returns a Custom DC SV in the response."""
+    query = "Average annual wages in Europe"
+    url = f'{CDC_AUTOPUSH_URL}/api/explore/detect-and-fulfill?q={query}'
+    response = requests.post(url, json={
+        'contextHistory': [],
+        'dc': '',
+    }).json()
+    svs = response.get('context', [{}])[0].get('svs', [])
+    assert 'average_annual_wage' in svs, 'Custom SV (average_annual_wage) not in detect-and-fulfill response'
```

Comment:
@pradh - this only tests that a custom SV was returned and is not testing NL functionality exhaustively like `explore_test`. Should it? Also, any other tests that may be useful as a NL sanity check?

Reply:
I think it should, and I feel we should use the exact same tests, albeit with a different URL?

Reply:
SG. Done.